src/components/editMeasure/details/MeasureReferences/MeasureReferences.tsx

```diff
@@ -115,12 +117,21 @@ const MeasureReferences = (props: MeasureReferencesProps) => {
       copiedMetaData.hasOwnProperty("references") &&
       Array.isArray(copiedMetaData.references)
     ) {
-      // if it does exist we push to it
-      copiedMetaData.references.push(values);
-      copiedMetaData.references = sortByTypeThenReferences(
-        copiedMetaData.references
-      );
-      // copiedMetaData.references.sort();
+      if (!selectedReference) {
+        // if it does exist we push to it
+        copiedMetaData.references.push(values);
+        copiedMetaData.references = sortByTypeThenReferences(
+          copiedMetaData.references
+        );
+        // copiedMetaData.references.sort();
```

Comment:
Remove dead code or note when to re-enable.

Reply:
removed